apis/config/v1alpha1/vault_config_types.go

```diff
@@ -39,6 +39,9 @@ type VaultConfigSpec struct {
 	// Server is the url of the Vault server, e.g. "https://vault.acme.org"
 	Server string `json:"server"`
 
+	// Namesoace is the Namespace of vault on which to operate
+	Namespace string `json:"namespace,omitempty"`
```

Comment:
Since this is an optional field, shall we change the type to `*string`?

Reply:
Makes sense - done in 190a19f23c990bc3d44696d73935987283ae4ebc